packages/features/teams/components/TeamFeatures.tsx

```diff
@@ -0,0 +1,50 @@
+import { APP_NAME } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Icon } from "@calcom/ui";
+
+export const TeamFeatures = () => {
+  const { t } = useLocale();
+  const features = [
+    {
+      icon: <Icon.FiUsers className="h-5 w-5 text-red-500" />,
+      title: t("collective_scheduling"),
```

Comment:
this file should not exist. UpgradeScreen should include the feature column as a required prop @JeroenReumkens